tools/RAiDER/models/ecmwf.py

```diff
@@ -1,15 +1,16 @@
 import datetime
+
 import numpy as np
 from pyproj import CRS
 
-from RAiDER.utilFcns import round_date
```

Comment:
I'm not sure why this is removed, since it looks to me like the function is called in line 123 (in my version) in the _get_from_ecmwf function?

Reply:
I see what you mean. On that line I get a warning "Undefined name 'util'". I can add `import RAiDER.utilFcns as util` at the top of the file.